### DISCUSSION TITLE
Help to install license in Fireox

### DISCUSSION THREAD

**Discussion by simonchaconb** [aclap-dev/video-downloadhelper#1062] [Help and Q&A]
### Your question

Hello!

I need help to activate the license in the Firefox browser, it is true that when I bought and activated it the first time it was in Chrome but according to the activation email they do allow you to use the license in another browser and in fact I already had it active but I had to updated firefox and the license was removed.

When trying to activate it I get this message:

"
![Captura-licencia-downloaderhelper-firefox](https://github.com/user-attachments/assets/0deb7d5d-1ebe-4ce3-86ff-41f8097a7b3d)
The license is for Chrome but the extension build is for Firefox."

### Addon & CoApp version and details

version: 9.0.2.12
target: mozilla
channel: stable
lang: es-AR
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"mismatch":true,"key":"eec4fe347841455c","other_browser":"Chrome","this_browser":"Firefox"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:109.0) Gecko/20100101 Firefox/118.0
{
    "download_directory": "C:\\Users\\usuario\\Downloads\\Videos descargados App DownloadHelper",
    "view_options": {
        "all_tabs": true,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "open_count_store": 12
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I am just another user.

> according to the activation email they do allow you to use the license in another browser

You are entirely mistaken.  This has never been true.  You need to read again whatever it was that misled you to reach this conclusion.  There are 3 products:  VDH for Firefox, VDH for Chrome, VDH for Edge.  They are separate & in no way interchangeable.  Each one runs on its designated browser & not any other.  Any license you buy is for exactly one of those products & can be used only with that product.  The products have different features & different prices.  What you are trying to do will never succeed.

**Reply by simonchaconb** [reply]
Hello!

So by mistake, until the day I reported it, I was able to download videos from both browsers using the same license. If I'm reporting it, it's because I hadn't had this problem before. But if the purchase doesn't allow this (which is not specified in the conditions or in the email itself, to be honest), that's fine, I understand.

Anyway, I would ask that it be allowed to use multiple browsers in the future because I'm currently downloading videos from a website that, I don't know why, only lets me download images and videos from the Firefox browser. I actually reported this in this ticket in case you're interested in the subject and as you can see, that forced me to have the extension with the license active in both browsers: https://github.com/aclap-dev/video-downloadhelper/discussions/327

Greetings!
